runtime/doc/lsp.txt

```diff
@@ -1397,7 +1397,10 @@ format({opts})                                          *vim.lsp.buf.format()*
 
 hover()                                                  *vim.lsp.buf.hover()*
     Displays hover information about the symbol under the cursor in a floating
-    window. Calling the function twice will jump into the floating window.
+    window. The window will be dismissed on cursor move. Calling the function
+    twice will jump into the floating window. Once focused in the floating
+    window, the buffer content can be scrolled as usual and `q` can be used to
+    dismiss the window.
```

Comment:
```suggestion
    twice will jump into the floating window (thus with the default mappings, "KK" will open the hover window and focus it). In the floating window, all commands and mappings are available as usual, except that "q" dismisses the window. You can scroll the contents the same as you would any other buffer.
```

Reply:
Didn't know we had this convention. Maybe something to document in `:help dev`?